src/views/Dashboard/Customer/Home/MyTest/TestResult.vue

```diff
@@ -68,6 +68,13 @@
         @onClose="showModalRating = false"
       )
         template
+          ui-debio-rating(
+            :size="33"
+            :total-rating="5"
+            :with-reviewers="false"
+            :interactive="true"
```

Comment:
if you want to make this interactive true, just type `interactive` like this

```pug
ui-debio-rating(
  :size="33"
  :total-rating="5"
  :with-reviewers="false"
  interactive
  @input="getRating"
 )
```

Reply:
ok, done.